source/openapi-admin-v3.yaml

```diff
@@ -5603,7 +5688,24 @@ components:
         error_code:
           type: string
           description: The error type.
+    AtlasClusterConstructorWithRules:
+      allOf:
+        - $ref: "#/components/schemas/AtlasClusterConstructor"
```

Comment:
My response body did not match the `AtlasClusterConstructor` schema `config` object. My config only had `clusterName`, `readPreference`, and `wireProtocolEnabled`. It did not have `clusterName`, `clusterId`, `clusterType`, `groupName`, etc. My response body had a `sync` config outside of this config object - it wasn't in the Atlas cluster config. I'm not sure if this schema is used elsewhere, but it doesn't match what I got from the pull endpoint.